config/sql/se/Ebooks.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE `Ebooks` (
+  `EbookId` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Identifier` varchar(511) NOT NULL,
+  `Created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `Updated` timestamp NOT NULL,
+  `WwwFilesystemPath` varchar(511) NOT NULL,
+  `RepoFilesystemPath` varchar(511) NOT NULL,
+  `KindleCoverUrl` varchar(511) NULL,
+  `EpubUrl` varchar(511) NULL,
+  `AdvancedEpubUrl` varchar(511) NULL,
+  `KepubUrl` varchar(511) NULL,
+  `Azw3Url` varchar(511) NULL,
+  `DistCoverUrl` varchar(511) NULL,
+  `Title` varchar(255) NOT NULL,
+  `FullTitle` varchar(255) NULL,
+  `AlternateTitle` varchar(255) NULL,
+  `Description` varchar(511) NOT NULL,
+  `LongDescription` text NOT NULL,
+  `Language` varchar(10) NULL,
+  `WordCount` int(10) unsigned NOT NULL,
+  `ReadingEase` float NOT NULL,
+  `GitHubUrl` varchar(255) NULL,
+  `WikipediaUrl` varchar(255) NULL,
+  `Released` datetime NOT NULL,
+  `MetadataModified` datetime NOT NULL,
+  `TextSinglePageSizeNumber` varchar(10) NULL,
```

Comment:
This should be a float, it will be something like `1.2` and then concatenated with `TextSinglePageSizeUnit` to result in something like `1.2MB`.

Reply:
Sure, done. The existing `TextSinglePageSizeNumber` property in the `Ebook` class is a string formatted with `sprintf()` ([link](https://github.com/standardebooks/web/blob/ff8cc6554d04c0eed172329d9c9f5f33aefe8f02/lib/Ebook.php#L131)):

```php
$this->TextSinglePageSizeNumber = sprintf('%.1f', $bytes / pow(1024, $factor));
```

so now I'm calling

```
floatval($this->TextSinglePageSizeNumber)
```

when writing this to the DB, and it works fine. We can look closer at the PHP code in the next PR, but if you'd like to peek at a rough draft it's here:

https://github.com/colagrosso/se-web/blob/9595a5c65bdced6b7a508d1aacf7fb38039cc86b/lib/Ebook.php#L142

We could do it differently:
* Don't store `TextSinglePageSizeNumber` or `TextSinglePageSizeUnit` in the DB
* Instead, store a new field, `TextSinglePageSizeByteCount bigint unsigned`, with the raw byte count and populate `TextSinglePageSizeNumber` or `TextSinglePageSizeUnit` after reading

Reply:
Ah, that's a good idea. Let's do one column, `TextSinglePageByteCount`.

Reply:
Done